zone/command.cpp

```diff
@@ -6525,6 +6526,144 @@ void command_doanim(Client *c, const Seperator *sep)
 			c->DoAnim(atoi(sep->arg[1]),atoi(sep->arg[2]));
 }
 
+void command_editmassrespawn(Client* c, const Seperator* sep)
+{
+	if (strcasecmp(sep->arg[1], "usage") == 0) {
+		c->Message(Chat::White, "#editmassrespawn [exact_match: =]npc_type_name new_respawn_seconds (apply)");
+		return;
+	}
+
+	std::string search_npc_type;
+	if (sep->arg[1]) {
+		search_npc_type = sep->arg[1];
+	}
+
+	int change_respawn_seconds = 0;
+	if (sep->arg[2] && sep->IsNumber(2)) {
+		change_respawn_seconds = atoi(sep->arg[2]);
+	}
+
+	bool change_apply = false;
+	if (sep->arg[3] && strcasecmp(sep->arg[3], "apply") == 0) {
+		change_apply = true;
+	}
+
+	std::string search_encapsulator = "%";
+	if (search_npc_type[0] == '=') {
+
+		search_npc_type = search_npc_type.substr(1);
+		search_encapsulator = "";
+	}
+
+	std::string query = fmt::format(
+		SQL(
+			SELECT tc.id, ta.spawngroupID, ta.id, tc.name, ta.respawntime
```

Comment:
I'd prefer we don't alias these tables; makes readability hard

Reply:
I'll try it with table names..not sure if it will affect INNER JOIN performance.

As long as huffin is happy with functionaility, I can validate these changes locally.

Reply:
Aliases don't affect the query plan analyzer, you just have to use the full table name as a prefix to the join itself